src/cli.ts

```diff
@@ -1,7 +1,64 @@
-function main(packageRootDir: string) {
+import * as fs from 'fs';
+import meow from 'meow';
+import pify from 'pify';
+
+import {outputToUser} from './output';
+import {generatePackageTree} from './package-tree';
+
+const cli = meow({
+  help: `  
+    Usage
+      $ TODO <project>
+ 
+    Options
+      --help        Prints this help message.
+  `,
+  flags: {help: {type: 'boolean'}}
+});
+
+if (cli.input.length !== 1) {
+  console.error(`Error: should have 1 argument, but got ${cli.input.length}`);
+  cli.showHelp(1);
+  process.exit(1);
+} else {
+  run(cli.input[0]);
+}
+
+async function validNodePackage(path: string) {
+  try {
+    await pify(fs.stat)(path);
+  } catch (err) {
+    console.error(`${err}`);
```

Comment:
Just log the error, no need to wrap in a template literal.

Reply:
done